test/fixtures/users.yml

```
@@ -1,14 +1,14 @@
-one:
-  email: one@example.com
+john:
+  email: john@example.com
   encrypted_password: <%= Devise::Encryptor.digest(User, 'password') %>
-  name: John
+  name: john
```

```suggestion
  name: John
```

固有名詞なら大文字で始まりますね（これも英語の話）